src/twisted/words/test/test_domish.py

```diff
@@ -350,6 +350,22 @@ def test_namespaceWithWhitespace(self):
             self.elements[0].attributes, {(" bar baz ", "baz"): "quux"})
 
 
+    def test_attributesWithNamespaces(self):
+        """
+        Attributes with namespace are parsed correctly (#9730 regression test).
```

Comment:
1. "Correctly" is a bit of an anti-pattern in test docstrings.  For some style notes, see https://jml.io/pages/test-docstrings.html which we try to adhere to within Twisted.
2. I love the "regression test" note, but given that Twisted is almost certainly going to switch bugtrackers at some point, a link (https://twistedmatrix.com/trac/ticket/9730) is vasty preferable so that folks don't need to worry what ticket-numbering scheme we're talking about here.  (We already have the issue of github PR numbers versus trac ticket numbers.)